doc/lua_api.md

```diff
@@ -7876,8 +7876,7 @@ child will follow movement and rotation of that bone.
       whether `set_sky` accepts this format. Check the legacy format otherwise.
     * Passing no arguments resets the sky to its default values.
     * `sky_parameters` is a table with the following optional fields:
```

Comment:
> problem: currently you can either opt for a textured skybox or decide the fog color, you can't do both.

You can already have a textured skybox with a custom fog color?

https://github.com/minetest/minetest/blob/b3e76382fae07f11822eb46dc9102d619ce51d1e/doc/lua_api.md?plain=1#L7886

Reply:
Indeed, I wrote the wrong thing.